wiki/People/Tournament_Committee/en.md

```diff
@@ -1,3 +1,8 @@
+---
+tags:
+  - TC
```

Comment:
```suggestion
  - TC
  - Contest Committee
  - CC
```

Reply:
CC is being used for other tags (ex. Community Choice and Community Contributor) it is more popularized for these areas and would create confusion

Reply:
it's just a search tag